php/plan.sh

```diff
@@ -51,8 +60,13 @@ do_build() {
     --with-openssl="$(pkg_path_for openssl)" \
     --with-png-dir="$(pkg_path_for libpng)" \
     --with-xmlrpc \
-    --with-zlib="$(pkg_path_for zlib)"
-  make
+    --with-zlib="$(pkg_path_for zlib)" \
+    --enable-zip \
+    --with-libzip="$(pkg_path_for "core/libzip")" \
+    --with-bz2="$(pkg_path_for "core/bzip2-musl")" \
```

Comment:
Is there a need for the musl based bzip2? I don't think we want to mix musl and glibc, if possible.